pyaedt/edb_core/EDB_Data.py

```diff
@@ -635,13 +688,15 @@ def remove_layer(self, layername):
         lcNew = self._parent.edb.Cell.LayerCollection()
         newLayers.Reverse()
         if not lcNew.AddLayers(newLayers) or not self._parent.active_layout.SetLayerCollection(lcNew):
-            self.messenger.add_error_message('Failed to set new layers when updating stackup info')
+            self.messenger.add_error_message('Failed to set new layers when updating the stackup information.')
             return False
         self._update_edb_objects()
         return True
 
 
 class EDBPadProperties(object):
+    """EDBPadProperties class.""
```

Comment:
```suggestion
    """EDBPadProperties class."""
```

Reply:
@PipKat, this was causing the syntax error.  Funny how a single character can ruin your whole day.

One takeaway from this is to look above for what might be causing the issue.  It might be a change you made several lines up rather than where the compiler reports the syntax error.